Demo/MockVenmo/ViewController.swift

```diff
@@ -2,8 +2,13 @@ import UIKit
 
 class ViewController: UIViewController {
 
-    @IBAction func didTapSuccess(_ sender: UIButton) {
-        guard let successURL = AppSwitcher.successURL else { return }
+    @IBAction func didTapSuccessWithPaymentContext(_ sender: UIButton) {
+        guard let successURL = AppSwitcher.successURLWithPaymentContext else { return }
+        UIApplication.shared.open(successURL, options: [:], completionHandler: nil)
```

Comment:
This can just be `UIApplication.shared.open(successURL)` since we aren't passing any options and the handler is nil.